test/extended/cluster/config/master-vert.yaml

```diff
@@ -0,0 +1,60 @@
+provider: local
+ClusterLoader:
+  projects:
+    - num: 1
+      basename: cakephp-mysql
+      tuning: default
+      templates:
+        - num: 1
+          file: ./testdata/quickstarts/cakephp/cakephp-mysql.json
```

Comment:
Ahh, I think this should have been part of this PR.  Our library of fragments for tests is here:
https://github.com/openshift/svt/tree/master/openshift_scalability/content